tool/site/server/src/services/communication/live-signing-client.ts

```diff
@@ -19,7 +20,10 @@ export class LiveSigningClient implements SigningClient {
 
   async uploadSignatureRequest(prepareResponses: Parameters[]): Promise<any> {
     const baseUrl = this.getBaseUrl()
-    const url = `${baseUrl}/signaturerequest`
+    const stateJson = {prNumber: getPrNumber(CONFIG.basePath)}
+    const stateString = JSON.stringify(stateJson)
+    const state = Buffer.from(stateString, "utf-8").toString("base64")
```

Comment:
again my knowledge gap but why do we need the `base64` parsing

Reply:
it gets passed to a query string parameter